### ISSUE TITLE
Make forms optional instead of required

### ISSUE THREAD

**Issue by marcobiedermann** [contao/core#7664]
For better UX, I recommend to change the form fields from being mandatory fields to optional fields.
The default should be required.
http://uxmovement.com/forms/always-mark-optional-form-fields-not-required-ones/

**Comment by aschempp** [comment]
You can easily accomplish this already using CSS and the placeholder attribute.

**Comment by marcobiedermann** [comment]
Yes, but currently you can only enable that a form field is required. I would switch it around. Standart should be required and you can enable that the field is optional. This has a much better user experience.

**Comment by Aybee** [comment]
Nice article, nice thinking.

**Comment by grusch-it** [comment]
As far as I see there isn't any evidence for the assertions made in the article (e.g. studies)?

Since the articles doesn't give any evidence (e.g. studies), the only interesting thing would be including a translated placeholder for fields not marked as required like shown in the articles second screenshot.

**Comment by marcobiedermann** [comment]
@grusch-it This is a really good video, which shows some nice tops to increase the UX with forms:
http://youtu.be/ulJwW6zfR_o

**Comment by Anke** [comment]
+1 (makes sense)
